resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/filesystem/SymlinkTest.java

```diff
@@ -37,7 +38,7 @@ public class SymlinkTest extends FileSystemTest {
 	 */
 	private static final boolean SYMLINKS_ARE_FIRST_CLASS_FILES_OR_DIRECTORIES = Platform.OS_WIN32
 			.equals(Platform.getOS()) ? true : false;
-	private static String specialCharName = "���� ��� ���� �����"; //$NON-NLS-1$
+	private static String specialCharName = "���� ��� ���� �����"; //$NON-NLS-1$
```

Comment:
broken encoding

Reply:
You're right, it seems buggy in GitHub. I've checked my local history anyway and it seems I unintentionally changed the code. I just reverted it back to what it was before so the special characters (`äöüß ÄÖÜ àÀâÂ µ²³úá`) should be back in place. 

Thank you!